src/commands/min.js

```diff
@@ -24,7 +24,7 @@ exports.run = (client, message, args, callback) => {
         || !Number.isInteger(Number(args[0]))
     ) return callback("Das ist kein gültiger positiver 64Bit Integer...");
 
-    message.channel.send("Moralisch vertretbares Alter: " + ((Number(args[0]) / 2) + 7));
+    message.channel.send("Moralisch vertretbares Alter: " + (Number(args[0]) === 69 ? "heh" : (Number(args[0]) / 2) + 7));
```

Comment:
Da ist ne Klammer zu viel du Dösbaddel

Reply:
:facepalm:

Reply:
Da is gar keine klammer zu viel

Reply:
@tobi6112 gott bist du duhm

Reply:
> @tobi6112 gott bist du duhm

Solche Freveleien sind gegen unseren Code of Conduct. Bitte unterlasse in Zukunft so herabwürdigende Kommentare.